source/OrderedTypes/PredicativeLFP.lagda

```diff
@@ -0,0 +1,1704 @@
+           (↓-is-sup a) (join-is-lub-of L (α i , ↓ᴮ-inclusion L β a ∘ ⌞ s ⌟))
+       trunc-u : (Ǝ i ꞉ I₂ , α i is-a-small-cover-of ↓ᴮ L β a) holds
+               → b ∈ Small-𝓘nd
+       trunc-u = ∥∥-rec (holds-is-prop (Small-𝓘nd b)) u
+
+   Small-𝓘nd-⊆-𝓘nd : Small-𝓘nd ⊆ 𝓘nd
+   Small-𝓘nd-⊆-𝓘nd = Small-𝓘nd-Initial 𝓘nd is-small-c-cl is-small-ϕ-cl
+    where
+     is-small-c-cl : (i : I₁)
+                   → Y i ⊆ 𝓘nd
+                   → (b : B)
+                   → (b , Y i) ∈ R
+                   → b ∈ 𝓘nd
+     is-small-c-cl i C b r =
+      𝓘nd-is-c-closed (Y i) C b
+                      (is-small-pres-backward b (Y i) ∣ (i , (λ _ → id) , r) ∣)
+     is-small-ϕ-cl : (i : I₂)
+                   → (m : α i → B)
+                   → (b : B)
+                   → small-ϕ b (⋁ (α i , β ∘ m))
+                   → ((x : B) → x ≤ᴮ (⋁ (α i , β ∘ m)) → x ∈ 𝓘nd)
+                   → b ∈ 𝓘nd
+     is-small-ϕ-cl i m b s C =
+      𝓘nd-is-ϕ-closed (⋁ (α i , β ∘ m)) b
+                      (ϕ-is-small-forward (⋁ (α i , β ∘ m)) b s) C
+
+   𝓘nd-is-small : (b : B) → (b ∈ 𝓘nd) is 𝓥 small
+   𝓘nd-is-small b = (b ∈ Small-𝓘nd , small-𝓘nd-equiv-𝓘nd)
+    where
+     small-𝓘nd-equiv-𝓘nd : b ∈ Small-𝓘nd ≃ b ∈ 𝓘nd
+     small-𝓘nd-equiv-𝓘nd = logically-equivalent-props-are-equivalent
+                            (holds-is-prop (Small-𝓘nd b))
+                            (holds-is-prop (𝓘nd b))
+                            (Small-𝓘nd-⊆-𝓘nd b)
+                            (𝓘nd-⊆-Small-𝓘nd b)
+
+\end{code}
+
+As a corollary of the above result we get a predicative version of the least
+fixed point theorem. Notice that we are assuming our lattice is
+small-presented and that we have a bounded ϕ that corresponds to our
+monotone map. This is the most general statement that can be made but we are
+actively exploring other, cleaner, formulations. In particular see the below
+notion of density which makes no mention of a particular inductive definition.
+
+Least fixed point theorem:
+Given a 𝓥-sup-lattice L with a 𝓥-presentation and a monotone
+endomap f : L → L. If there exists a bounded abstract inductive definition
+ϕ such that f ＝ Γ ϕ, then f has a least fixed point.
+
+\begin{code}
+
+module _
+        {𝓤 𝓦 𝓥 : Universe}
+        {B : 𝓥  ̇}
+        (L : Sup-Lattice 𝓤 𝓦 𝓥)
+        (β : B → ⟨ L ⟩)
+        (h : is-small-basis L β)
+       where
+
+ private
+  _≤_ = order-of L
+  ⋁_ = join-of L
+ 
+ open local-inductive-definitions L β h
+ open bounded-inductive-definitions L β h
+ open small-presentation-of-lattice L β h
+
+ module QITs-exists-for-all-ϕ
+         (ind-e : (ϕ : 𝓟 {𝓤 ⊔ 𝓥} (B × ⟨ L ⟩))
+                → inductively-generated-subset-exists L β h ϕ)
+         (ind'-e : (ϕ : 𝓟 {𝓤 ⊔ 𝓥} (B × ⟨ L ⟩))
+                 → (bnd : is-bounded ϕ)
+                 → (small-pres : has-small-presentation)
+                 → small-QIT-from-bounded-and-small-presentation.inductively-generated-small-subset-exists L β h small-pres ϕ bnd)
```

Comment:
Just do `open small-QIT-from-bounded-and-small-presentation L β h small-pres ϕ bnd` before the module since you pass those arguments anywhere?